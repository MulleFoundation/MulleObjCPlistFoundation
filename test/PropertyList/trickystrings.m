//
//  main.m
//  archiver-test
//
//  Created by Nat! on 19.04.16.
//  Copyright Â© 2016 Mulle kybernetiK. All rights reserved.
//
#ifdef __MULLE_OBJC__
#import <MulleObjCPlistFoundation/MulleObjCPlistFoundation.h>
#import <MulleObjCOSFoundation/MulleObjCOSFoundation.h>
#else
# import <Foundation/Foundation.h>
#endif


int   main( int argc, const char * argv[])
{
   NSData                 *data;
   NSDictionary           *dict;
   NSPropertyListFormat   format;

   data   = [[NSFileHandle fileHandleWithStandardInput] readDataToEndOfFile];
   format = NSPropertyListOpenStepFormat;
   dict   = [NSPropertyListSerialization propertyListWithData:data
                                                      options:0
                                                       format:&format
                                                        error:NULL];

   mulle_printf( "%@\n", dict);
   return( 0);
}
