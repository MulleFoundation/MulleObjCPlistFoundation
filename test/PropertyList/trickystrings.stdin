{
   "›" = 1;
   "x" = ".  ";
   "crs" = "KK\r\r";
}
