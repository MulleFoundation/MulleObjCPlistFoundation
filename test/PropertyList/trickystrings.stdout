{
    crs = "KK\r\r";
    x = ".  ";
    "›" = 1;
}
